ingestion/src/metadata/profiler/orm/functions/md5.py

```diff
@@ -33,7 +31,12 @@ def _(element, compiler, **kw):
     return f"MD5({compiler.process(element.clauses, **kw)})"
 
 
-@compiles(MD5, Dialects.MSSQL)
+@compiles(MD5, PythonDialects.MSSQL.value)
 def _(element, compiler, **kw):
     # TODO requires separate where clauses for each table
     return f"CONVERT(VARCHAR(8), HashBytes('MD5', {compiler.process(element.clauses, **kw)}), 2)"
+
+
+@compiles(MD5, PythonDialects.BigQuery.value)
```

Comment:
Should we check for all the major ones at least to see if the default way the md5 function is compiled works as intented?

Reply:
I think we only support it for 10 connectors, we can test it for all 10
- Trino
- Athena
- MySQL
- Hana
- Oracle